src/components/InitSetConfig.jsx

```diff
@@ -19,28 +20,24 @@ const Item = styled(Paper)(({ theme }) => ({
 }));
 
 function InitSetConfig() {
-  const { walking, kinematic } = useContext(WalkContext);
+  const { kinematic, walking } = useContext(WalkContext);
 
-  const configPublisher = usePublisher();
-  const logger = useLogger();
+  const handlePublish = () => {
+    const client = new aruku_interfaces.ConfigClient(`${import.meta.env.GRPC_WEB_API_URL}`, null, null);
```

Comment:
also this, if GRPC_WEB_API_URL use in multiple files, you can define it as global variable. please check the others too